versions/3.0.4.md

```diff
@@ -1726,9 +1726,13 @@ Given a name of `example` and a solid red 2x2-pixel PNG for `icon`, this
 would produce a request body of:
 
 ```urlencoded
-name=example&icon=iVBORw0KGgoAAAANSUhEUgAAAAIAAAACCAYAAABytg0kAAAAEklEQVQIW2P8z8AARAwMjDAGACwBA/+8RVWvAAAAAElFTk
+name=example&icon=iVBORw0KGgoAAAANSUhEUgAAAAIAAAACCAIAAAD91JpzAAAABGdBTUEAALGPC%2FxhBQAAADhlWElmTU0AKgAAAAgAAYdpAAQAAAABAAAAGgAAAAAAAqACAAQAAAABAAAAAqADAAQAAAABAAAAAgAAAADO0J6QAAAAEElEQVQIHWP8zwACTGCSAQANHQEDqtPptQAAAABJRU5ErkJggg%3D%3D
 ```
 
+Note that this base64-encoded value had to be futher percent-encoded, replacing `/` with `%2F` and each of two final `=` padding characters with `%3D`.
```

Comment:
> replacing `/` with `%2F`

That's why [base64url](https://datatracker.ietf.org/doc/html/rfc4648#section-5) was invented - I wonder why isn't used more often in places where it makes sense.

Reply:
Yes, it's allowed and used in an example in 3.1.  Oddly, the use of `=` for padding is not changed, so you still end up needing to do percent-encoding :-/